packages/core/src/Table/stories/TableSection/TableSection.stories.mdx

```diff
@@ -67,3 +68,19 @@ the section content.
 <Canvas>
   <Story name="Editable Table" story={EditableStory} />
 </Canvas>
+
+### Table Filter
```

Comment:
One one hand, I would expect a sample that's _all about_ filtering to be under `useHvFilters`... on the other hand it does contain an `HvTableSection`... 🤷

Reply:
I understand. I can add a link in the `useHvFilters` page to let the users know we have this sample. It just seems a bit weird (to me) to have this sample with a completely different styling inside `useHvFilters`.